AlphaWallet/KeyManagement/LegacyFileBasedKeystore.swift

```diff
@@ -14,20 +14,21 @@ enum FileBasedKeystoreError: LocalizedError {
 class LegacyFileBasedKeystore {
     private let keychain: KeychainSwift
     private let datadir = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)[0]
+    ///LegacyFileBasedKeystore keystore
     private let keyStore: KeyStore
-    private let analyticsCoordinator: AnalyticsCoordinator
-
+    ///Etherkeystore
+    private let keystore: Keystore
```

Comment:
Haha. Shall we just rename `keystore` to `etherkeystore` ? Not perfect, but better given the unfortunately type names :)

Reply:
Then we can store out `Keystore` and `KeyStore` in the future.